### ISSUE TITLE
Downloading of virtio-win.iso Fails

### ISSUE THREAD

**Issue by suuuehgi** [quickemu-project/quickemu#646]
#### Expected behaviour

Work

#### Actual behaviour

```log
02/12 11:34:02 [ERROR] CUID#7 - Download aborted. URI=https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/stable-virtio/virtio-win.iso
Exception: [AbstractCommand.cc:351] errorCode=1 URI=https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/stable-virtio/virtio-win.iso
  -> [SocketCore.cc:1018] errorCode=1 SSL/TLS handshake failure:  `not signed by known authorities or invalid' `issuer is not known'
```

#### Steps to reproduce the behaviour

`quickget windows 10`

#### Linux Distribution & Kernel

```bash
> uname -a
Linux hostname 6.1.9-arch1-2 #1 SMP PREEMPT_DYNAMIC Fri, 03 Feb 2023 18:49:53 +0000 x86_64 GNU/Linux
```

The issue can be circumvented by adding `--check-certificate=false` to `aria2c` on line 706 of `/usr/bin/quickget` but this obviously comes with risks.

**Comment by ntzb** [comment]
same, the suggested workaround helps with getting the needed files.

**Comment by john-peterson** [comment]
Do you know how to skip the iso download. Snail pace today. FIVE HOURS for Windows iso. And I just need the config files and firmware dumps. I downloaded all images from vagrant anyway

**Comment by bcmryan** [comment]
I can confirm that this issue is still ongoing. In the end, I just downloaded the latest version from https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/stable-virtio/virtio-win.iso (the link is given in the error message) and manually created a .conf file, e.g. `windows-11-22H2.conf` as

```
guest_os="windows"
disk_img="windows-11/disk.qcow2"
iso="windows-11/windows-11.iso"
fixed_iso="windows-11/virtio-win.iso"
tpm="on"
secureboot="on"
```

Note that you will either need to rename the resulting file (e.g. `virtio-win-0.1.240.iso`) to `virtio-win.iso` or you will need to edit the .conf file so that the name of the `fixed_iso` is accurate.

Per usual, this will be able to be launched as `quickemu --vm windows-11-22H2.conf`.

**Comment by flexiondotorg** [comment]
Fixed via #836
